openstudiocore/src/model/HeatPumpWaterToWaterEquationFitCooling.hpp

```diff
@@ -60,16 +62,24 @@ class MODEL_API HeatPumpWaterToWaterEquationFitCooling : public WaterToWaterComp
   //@{
 
   /** In EnergyPlus 8.7.0 and above this field maps to the EnergyPlus field named "Reference Load Side Flow Rate" **/
-  double ratedLoadSideFlowRate() const;
+  boost::optional<double> ratedLoadSideFlowRate() const;
```

Comment:
Good point.  The only thing I can think of is that this is a pretty uncommon object, so a breaking change wouldn't affect many people (I'm guessing?)